docs/getting-started.mdx

```diff
@@ -8,46 +8,43 @@ import { AssetsProvider } from '@site/src/components/assets-context';
 import DownloadCard from '@site/src/components/download-card';
 import DownloadCardStyles from '@site/src/css/download-card.module.css';
 
+## Share your network resources within minutes
+
+Easily share private network resources with public internet users, securely, without having to alter any of your local network configuration.
+You can share resource using [NetFoundry's cloud-hosted zrok](https://myzrok.io/),
+or follow our [self-hosted guide](/docs/category/self-hosting/) to manage your own instance.
+
+* Download the binary for your platform [here](https://github.com/openziti/zrok/releases/latest)
+* Sign up for a myzrok.io account [here](https://myzrok.io/) to enable your personal share token
+* Run `zrok enable <your_token>` to enable your shell environment for sharing
+* Run `zrok share public localhost:8080` to start your first share!
+
+![zrok share public](images/zrok_share_public.png)
+
+Access your share from anywhere!
+
+```
+$ curl -s https://dslno640nct4.share.zrok.io | head
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="theme-color" content="#000000"/>
+    <meta name="description" content="zrok ui"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+```
+
+zrok provides several different share types, including:
+
+* [public shares](./concepts/sharing-public.mdx) for [web services](./concepts/http.md) or [files](./concepts/files.md)
+* [private shares for web services or files](./concepts/sharing-private.mdx)
+* [reserved shares](./concepts/sharing-reserved.md)
+* [tcpTunnel shares](./concepts/tunnels.md)
```

Comment:
It makes sense to me to present specific backend modes in the context of the share types to which they apply, e.g., `tcpTunnel` is a private backend mode. That's how they're broken down in the separate private/public concept docs which serve as a summary and index of sharing modes with direct links to the docs that further expand on each, i.e., the private share concept doc has an index of applicable sharing modes that link to the most appropriate pages, like the tunnels concept doc.

Reply:
Part of why I wanted to present these right out front is that a very small percentage of users ever use anything beyond the public web proxy, and I'd like to see if we can immediately answer a user's question "can I host a TCP service with this?" Right now if you scan all topics within our docs, you can't find the answer to that, it's hidden in the private share documentation. I'm open to approach it differently but I'm hoping to make some of the more advanced features that zrok has more apparent so that users will pick them up and use them.

Reply:
Yes, maybe we can surface those features, make them easier to discover. Maybe an FAQ or "How do I ____?" sidebar?